src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -10183,12 +10215,26 @@ static bool isCandidateUnique(ref MethodSymbol? method, MethodSymbol candidate)
                         continue;
                     }
 
+                    methods.Add(substituted);
+                }
+
+                if (!OverloadResolution.FilterMethodsForUniqueSignature(methods))
```

Comment:
>methods [](http://example.com/codeflow?start=72&length=7)

I think that we want to get a set of ```MemberResolutionResult<MethodSymbol>``` back. Otherwise, how would we know if a candidate is a valid candidate for an expanded form?